sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -76,11 +76,11 @@ def _from_generated(cls, generated):
             created_on=generated.created_on,
             digest=generated.digest,
             last_updated_on=generated.last_updated_on,
-            manifest_properties=generated.manifest_properties,
             operating_system=generated.operating_system,
-            registry_artifacts=generated.registry_artifacts,
+            references=generated.references,
```

Comment:
Just checking - should this be? 

```suggestion
            references=generated.registry_artifacts,
```